main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        users = json.loads(file.read())
+        for nickname, params in users.items():
+            params_race = params["race"]
+            race, _ = Race.objects.get_or_create(
+                name=params_race["name"],
+                description=params_race["description"]
+            )
```

Comment:
Ensure that the `params_race` dictionary contains both 'name' and 'description' keys to avoid KeyError exceptions. Consider using `params_race.get('name')` and `params_race.get('description')` with default values or handling missing keys appropriately.